docs/api_internal.rst

```diff
@@ -40,11 +40,56 @@ GPIO Zero also provides several "internal" devices which represent facilities
 provided by the operating system itself. These can be used to react to things
 like the time of day, or whether a server is available on the network.
 
-.. warning::
+These devices provide an API similar to and compatible with GPIO devices so that
+internal device events can trigger changes to GPIO output devices the way input
+devices can. In the same way a :class:`~gpiozero.Button` object is *active* when
+it's pressed, and can be used to trigger other devices when its state changes,
+a :class:`TimeOfDay` object is *active* during a particular time period.
 
-    These devices are experimental and their API is not yet considered stable.
-    We welcome any comments from testers, especially regarding new "internal
-    devices" that you'd find useful!
+Consider the following code in which a :class:`~gpiozero.Button` object is used
+to control an :class:`~gpiozero.LED` object::
+
+    from gpiozero import LED, Button
+    from signal import pause
+
+    led = LED(2)
+    btn = Button(3)
+
+    btn.when_pressed = led.on
+    btn.when_released = led.off
+
+    pause()
+
+Now consider the following example in which a :class:`TimeOfDay` object is used
+to control an :class:`~gpiozero.LED` using the same method::
+
+    from gpiozero import LED, TimeOfDay
+    from datetime import time
+    from signal import pause
+
+    led = LED(2)
+    tod = TimeOfDay(time(9), time(10))
+
+    tod.when_activated = led.on
+    tod.when_deactivated = led.off
+
+    pause()
+
+Here, rather than the LED being controlled by the press of a button, it's
+controlled by the time. When the time reaches 09:00AM, the LED comes on, and at
+10:00AM it goes off.
+
+Like the :class:`~gpiozero.Button` object, internal devices like the
+:class:`TimeOfDay` object has :attr:`~TimeOfDay.value`,
+:attr:`~TimeOfDay.values`, :attr:`~TimeOfDay.is_active`,
+:attr:`~TimeOfDay.when_activated` and :attr:`~TimeOfDay.when_deactivated`
+attributes, so alternative methods using the other paradigms would also work.
+
+.. note::
+    Note that although the constructor parameter ``pin_factory`` is available
+    for internal devices, and is required to be valid, the pin factory chosen
+    will not make any difference. Reading a remote Pi's CPU temperature, for
+    example, is not currently possible.
```

Comment:
Not *precisely* true, but probably true enough for the purposes of the documentation. The pedant in me, however, wants to point out that it *will* make a difference to whatever calls the pin-factory's `ticks` method (which `EventsMixin` does for the purpose of figuring out the timing of events to report to handlers, like `Button.hold_time`). However, I don't *think* anything in internal devices actually uses the ticks value beyond passing it around because it has to. So, yeah, don't bother correcting this my inner pedant!

Reply:
"will not make any *practical* difference" is probably more accurate - and if at any point this changes we can update the note.